### PR TITLE
(maint) Improve readability

### PR DESCRIPTION
**Pull request by mwaggett**
see individual commit messages for details

### CODE REVIEW

File: integration/tests/user_scenario/basic_workflow/single_env_purge_unmanaged_modules.rb [hunk 1]
```diff
@@ -6,8 +6,8 @@
 #Init
 master_certname = on(master, puppet('config', 'print', 'certname')).stdout.rstrip
 moduledir = on(master, puppet('config', 'print', 'environmentpath')).stdout.strip + '/production/modules'
-git_environments_path = '/root/environments'
-last_commit = git_last_commit(master, git_environments_path)
+git_remote_environments_path = '/root/environments'
```
Comment (justinstoller) [inline integration/tests/user_scenario/basic_workflow/single_env_purge_unmanaged_modules.rb:9]: Did "git" imply "remote" here before? I do like "remote" better but I'm wondering if we need "git" as part of the name?

File: integration/tests/user_scenario/basic_workflow/single_env_purge_unmanaged_modules.rb [hunk 2]
```diff
@@ -76,7 +76,7 @@ class { 'motd':
 end
 
 step 'Use r10k to Purge Unmanaged Modules'
-on(master, "#{r10k_fqp} puppetfile purge -v --puppetfile #{puppet_file_path} --moduledir #{moduledir}")
+on(master, "#{r10k_fqp} puppetfile purge --puppetfile #{remote_puppetfile_path} --moduledir #{moduledir} --verbose")
```
Comment (justinstoller) [inline integration/tests/user_scenario/basic_workflow/single_env_purge_unmanaged_modules.rb:79]: Seems like this change exposes the fact that we're using the Puppetfile in the "remote" to purge content in the "local" environment. Should we move to using the "local" Puppetfile in that environment?

Comment (mwaggett) [reply integration/tests/user_scenario/basic_workflow/single_env_purge_unmanaged_modules.rb:79]: yep, this was part of the fun debugging I was doing last night, but I was going to make functional changes in a separate PR.

I haven't quite gotten to what the actual Puppetfile should be. I found this documentation that contradicts some of the work you did recently, I think: https://github.com/puppetlabs/r10k/blob/main/doc/puppetfile.mkd#commands

File: integration/tests/user_scenario/basic_workflow/single_env_purge_unmanaged_modules.rb (hunk 1 above)
Comment (mwaggett) [reply integration/tests/user_scenario/basic_workflow/single_env_purge_unmanaged_modules.rb:9]: yeah I went back and forth on whether to leave "git" in or not. I opted for extra clarity around the fact that this is used to simulate committing to a control repo, but I can remove the "git" part if you don't like it.

File: integration/tests/user_scenario/basic_workflow/single_env_purge_unmanaged_modules.rb (hunk 2 above)
Comment (justinstoller) [reply integration/tests/user_scenario/basic_workflow/single_env_purge_unmanaged_modules.rb:79]: what contradicts recent work?

Comment (mwaggett) [reply integration/tests/user_scenario/basic_workflow/single_env_purge_unmanaged_modules.rb:79]: > Puppetfile subcommands assume that the Puppetfile to operate on is in the current working directory and modules should be installed in the 'modules' directory relative to the current working directory.

Comment (justinstoller) [reply integration/tests/user_scenario/basic_workflow/single_env_purge_unmanaged_modules.rb:79]: how's that contradict recent work?

Comment (mwaggett) [reply integration/tests/user_scenario/basic_workflow/single_env_purge_unmanaged_modules.rb:79]: idk this implied to me that the Puppetfile could be anywhere: https://github.com/puppetlabs/r10k/commit/46fb24e6a00ab81ae6c8dc0136dd6d92d7fa37a6

but not really planning to figure that out right now, if I don't have to.

Comment (mwaggett) [reply integration/tests/user_scenario/basic_workflow/single_env_purge_unmanaged_modules.rb:79]: okay, turns out I read things wrong / didn't read some things at all 😅 and what Puppetfile paths we allow for the `purge` command hasn't changed, so ignore all that 🙃

**Comment by mwaggett** [comment]
@justinstoller Are there any changes you want me to make or can this go in?

**Comment by justinstoller** [comment]
Did you want this to change this to use the "local" Puppetfile rather than the "remote" one? I think it'd be an improvement, but I wouldn't block fixing up CI on it if this work is blocking CI.

**Comment by mwaggett** [comment]
I'm planning to make that change in a separate PR. I'm trying to make small, incremental changes as I debug this.

**Review by justinstoller — APPROVED** [review]

**Comment by justinstoller** [comment]
Would you like help debugging it?

**Comment by mwaggett** [comment]
I think I'm alright for now, but thank you! I'm pretty sure I know the changes I need to make and I've had them since the day before yesterday, but weird stuff keeps happening in my local testing that I'm trying to understand. 🙃 
We can chat in standup and in channel if I need help. 😊

**Pull request merged**